openapi/platform/platform.yaml

```diff
@@ -3919,6 +3919,938 @@ paths:
+      security:
+        - OAuth2:
+            - sales.proposals
+      tags:
+        - Proposals
+      x-lifecycle:
+        status: proposed
+    post:
+      security:
+        - OAuth2:
+            - sales.proposals
+      x-lifecycle:
+        status: proposed
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
+        Create a proposal template that can be used by Proposal Builder.
+      summary: Create Proposal Template
+      operationId: post-proposals
+      parameters:
+        - schema:
+            type: string
+            example: Bearer <Access Token>
+            pattern: ^Bearer\s\S+
+          in: header
+          name: Authorization
+          description: A Bearer access token to identify the user the app is acting on behalf of. See the Authorization guide for details.
+          required: true
+        - schema:
+            type: string
+          in: header
+          name: Accept-Language
+        - schema:
+            type: string
+            default: application/vnd.api+json
+            enum:
+              - application/vnd.api+json
+          in: header
+          name: Content-Type
+          required: true
+          description: Indicates the format of the request body being sent. In most cases you will want `application/vnd.api+json`
+      responses:
+        '201':
+          description: Created
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/proposalTemplates'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+                        format: uri
+                        description: The address of the newly created proposal
+          headers:
+            Location:
+              schema:
+                type: string
+                format: uri
+              description: The address of the newly created proposal
+      requestBody:
+        content:
+          application/vnd.api+json:
+            schema:
+              type: object
+              properties:
+                data:
+                  $ref: '#/components/schemas/proposalTemplates'
+      tags:
+        - Proposals
+    options:
+      operationId: options-proposalTemplates
+      summary: List valid HTTP verbs for /proposalTemplates
+      description: 'Used solely for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) the OPTIONS request returns the list of possible HTTP methods and other headers that browsers use to protect user''s security. You should not call this operation directly. '
+      responses:
+        '204':
+          description: No Content
+      tags:
+        - Options
+        - Proposals
+    parameters: []
+  '/proposalTemplates/{id}':
```

Comment:
It appears that something went wrong with the merge. Some of the proposal stuff is now listed twice in the yaml. When looking at this PR there should not be any changes related to proposals.

Reply:
Fixed the merge issue.